doc/source/user/problem_method.rst

```diff
@@ -518,12 +518,27 @@ fields
 
 particles
 ---------
+
+For a given particle type to be deposited to the total density field,
+it must be part of the ``"is_gravitating"`` group, and must have
+exactly one of the following: an attribute called ``"mass"``, a
+constant called ``"mass"``, an attribute called ``"density"``, or
+a constant called ``"density"``.
+
```

Comment:
The next two paragraphs appear at odds with one another. For the 'mass' case the density of the particle in the cell is found. However for the 'density' case it is something different. In the 'density' case it looks like the density is being increased to account for cells which are refined (in a unigrid run nothing changes). It seems to me that this will only work if the 'density' is the mass divided by the volume of the root grid cell? Is that correct?